kie-server-parent/kie-server-services/kie-server-services-dmn/src/main/java/org/kie/server/services/dmn/ModelEvaluatorServiceBase.java

```diff
@@ -204,8 +217,128 @@ public static DMNItemDefinitionInfo itemDefinitionToInfo(ItemDefinition itemDef)
         }
     }
 
+    public Response evaluateModel(String containerId, String modelId, String contextPayload, boolean asDmnResult, String decisionServiceId) {
```

Comment:
Do you plan to support also JMS, or is it just REST service only?

Reply:
REST only.
I guess this can be reworked with a later RFE for JMS too, but only if requested.